Shared/Dtos/FromClient/UpdateUserDto.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace Shared.Dtos.FromClient;
+
+public class UpdateUserDto
+{
+    public string UserEmail { get; set; } = null!;
```

Comment:
it would be good to have an [EmailAddress] validation tag in here, and also some length constraints for the username, as we do for the other dto